components/webui/imports/api/ingestion/server/publications.js

```diff
@@ -99,8 +103,13 @@ const refreshCompressionJobs = async () => {
             job._id
         ));
 
+    const previousUpdateDate = lastUpdateDate;
+    lastUpdateDate = new Date().toISOString()
+        .slice(0, CONST_FOR_DATE_FORMAT)
+        .replace("T", " ");
```

Comment:
i can't be sure how fast computers are / there're any optimizations done by the JS VM, but we might have a delay in getting this current time.

i think a better approach to get the last updated time, which also handles a situation where the Meteor server's time going out sync with the MySQL server, is to `SELECT UNIX_TIMESTAMP()` as we perform the query in `getCompressionJobs`.